_posts/2017-09-17-this-week-in-rustfest-11-the-countdown.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "This Week in RustFest 11: Keynote"
+description: "Keynote, childcare, and other notes."
+authors:
+  - hoverbear
+  - dns2utf8
+---
+
+This week we are happy to announce our keynote for RustFest! We also exhausted our ticket sales, and explored the possibility of having childcare for the event.
+
+## Keynote
+
+This year for RustFest we're happy to welcome Alex Crichton to hold the stage. Alex is a member of the Rust Core Team who has worked at Mozilla on Rust for the past four years. He is also a member of the Tokio Core Team, one of the primary authors of Cargo, helps maintain the standard library, and likes to ensure that engine of Rust never stops.
+
+So what will he teach us about?
+
+![Alex Teaser: How we hit 88mph](http://zurich.rustfest.eu/assets/zurich/social/acrichto.png)
+
+> Async I/O has forever been a hot topic of discussion in Rust, but over the past year we’ve seen some significant advances in this domain with the futures crate an the Tokio project. We’ll start off by taking a look at where we are today with the async I/O ecosystem in Rust, highlighting a number of the features we’ve added in both the libraries and the language over the past year. We’ll then take a deep dive into Tokio itself and learn how it’s just so darn fast and capitalizes on some of Rust’s greatest strengths.
+
+## Childcare?
+
+We may have found a childcare provider for the duration of the conference. Before we go ahead and book we'd like to know if this service would be used. If you're attending the conference and would be interested in using a childcare service organized by the conference please send us an email at [team@rustfest.eu](mailto:team@rustfest.eu).
+
+## Student Tickets Exhausted
+
+This week we ran out of our allocated suppply of student tickets. This is both amazing and awful. It's amazing because it means we have lots of fantastic students to welcome to the conference. It's awful because it means we can't offer more tickets at this time. We can't wait to meet all of our studious attendees in Zürich!
+
+## Impl Days are comming
+
+The impl days are a place to learn, meet, share and hack with friendly people from the rust community.
+In the wo days (Monday 2017-10-02 and Thuesday 2017-10-03) after the conference we are going to work on ervery thing in the rust ecosystem from the compiler to crates.
+With the support of our friends at [mozilla](https://mozilla.org) we can provide some drinks and some snacks during the days. 
+However we will not organise catering, because there are many good places for food to go to or buy foot to the room in the area. 
```

Comment:
`However we will not organise catering, because there are many good places for food to go to or buy foot to the room in the area.` -> `However there will not be catering, there are many good places to get food in the area.`